### ISSUE TITLE
Source code of tiny_mce_popup.js is obfuscated and there's no non-obfuscated version

### ISSUE THREAD

**Issue by carlosmintfan** [barryvdh/laravel-elfinder#332]
https://github.com/barryvdh/laravel-elfinder/blob/master/resources/assets/js/tiny_mce_popup.js is mostly an one-liner. Why?

**Comment by barryvdh** [comment]
That is from tinyMCE directly. It's distributed minified so it's smaller. The source is in the 3.x branch of TinyMCE: https://github.com/tinymce/tinymce/blob/3.x/jscripts/tiny_mce/classes/Popup.js

You can compare it to the minified code there also: https://github.com/tinymce/tinymce/blob/3.x/jscripts/tiny_mce/tiny_mce_popup.js

I'm not 100% which version exactly, but I ran the last 3.x version and the version in my repo through a de-obfuscator to compare them, and it's just a IE fix that is missing in mine (left).

https://gist.github.com/barryvdh/78c9e4178e68c2d868b05cceb3f84561/revisions

![Image](https://github.com/user-attachments/assets/89ad3510-7f31-4c72-bbce-edcd4f69b3d6)

**Comment by carlosmintfan** [comment]
Ah, thank you for your answer. Note that you should follow the LGPL license then by making attribution and so on. It may be easier to use this MIT-licensed code: https://github.com/nao-pon/tinymceElfinder.

**Comment by barryvdh** [comment]
Yeah not sure if it even works with current tinyMCE. Added the exact copy + link to the source in https://github.com/barryvdh/laravel-elfinder/pull/334
